.github/workflows/case-data-export-prod.yml

```diff
@@ -28,11 +28,17 @@ jobs:
           sudo dpkg --purge mongodb-org-database-tools-extra
           sudo apt-get update
           sudo apt-get install -f -y --allow-downgrades mongodb-org=4.2.6 mongodb-org-server=4.2.6 mongodb-org-shell=4.2.6 mongodb-org-mongos=4.2.6 mongodb-org-tools=4.2.6 libcurl3
-
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Any reason we're using `python` here and `python3` below? Mostly a nit -- they're probably symlinked to the same thing in `setup-python`.

Reply:
ah I didnt notice, switched to pythn3 for consistency.